x/pricefeed/legacy/v0_16/migrate.go

```diff
@@ -16,9 +17,99 @@ func migrateParams(params v015pricefeed.Params) v016pricefeed.Params {
 			Active:     market.Active,
 		}
 	}
+
+	markets = addIbcMarkets(markets)
+
 	return v016pricefeed.Params{Markets: markets}
 }
 
+func addIbcMarkets(markets v016pricefeed.Markets) v016pricefeed.Markets {
+	var oracles []types.AccAddress
+
+	if len(markets) > 0 {
+		oracles = markets[0].Oracles
+	}
+
+	newMarkets := []v016pricefeed.Market{
+		{
+			MarketID:   "atom:usd",
+			BaseAsset:  "atom",
+			QuoteAsset: "usd",
+			Oracles:    oracles,
+			Active:     true,
+		},
+		{
+			MarketID:   "atom:usd:30",
```

Comment:
been wanting to ask this, what are the `:30` markets?

Reply:
TWAP, time weighted average price over 30 minutes

The ones without the `:30` suffix are spot prices

Reply:
awesome, thx!